### ISSUE TITLE
Settings to modify duration the video seeks when using the seek gesture

### ISSUE THREAD

**Issue by mukulrwt** [mpv-android/mpv-android#905]
Can we get settings to modify the seek precision? Like the seconds/milliseconds the video is seeked when using the seek gesture?

Currently, I can see that the video moves 2 seconds when seeking.

**Comment by sfan5** [comment]
just needs to be made configurable
https://github.com/mpv-android/mpv-android/blob/6b9e86c0cd793e633386dea39cfa635f109cadab/app/src/main/java/is/xyz/mpv/TouchGestures.kt#L80-L81

**Comment by mukulrwt** [comment]
Yes. But i think it also might need some other adjustments. It seems currently the time is rounded to nearest second. It would be great if we could remove the rounding off and do a frame-by-frame sweep.

This paired with touch gestures like pan and zoom with two fingers (I think there are already issues created for these) would be really nice.
